tools/roslaunch/scripts/roslaunch-check

```diff
@@ -50,12 +51,13 @@ def check_roslaunch_file(roslaunch_file, use_test_depends=False, ignore_unset_ar
         return "[%s]:\n\t%s"%(roslaunch_file,error_msg)
 
 def check_roslaunch_dir(roslaunch_dir, use_test_depends=False, ignore_unset_args=False):
+    roslaunch_files = []
+    for root, dirnames, filenames in os.walk(roslaunch_dir):
```

Comment:
This loop should probably ignore subdirectories starting with a `.` in order to e.g. not recursively search in `.git` / `.svn` directories (which in the latter case would contain the same launch files again).

Reply:
You're right.